src/components/EditCoursePage/CollapsibleCourseRunFields.jsx

```diff
@@ -74,6 +75,7 @@ const CollapsibleCourseRunFields = ({
           }
           placeholder="mm/dd/yyyy"
```

Comment:
Is that really our placeholder? That's not very non-US friendly. I'd assumed when I saw that on the page, that it was coming from the browser.

Obviously not your bug or problem, but humph.

Reply:
I think if we're pulling in `moment` anyway, we should be able to ask `moment` for a reasonable placeholder.